CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+## 2.6.0 (December 20, 2018)
+
+#### :rocket: New Feature
+* [lighter-react-scripts] add automatic src/scripts folder files transpile ([#49](https://github.com/lightingbeetle/lighter/pull/49), [#58](https://github.com/lightingbeetle/lighter/pull/58))
+* [lighter-react-scripts] add mdx config ([#59](https://github.com/lightingbeetle/lighter/pull/59), [#61](https://github.com/lightingbeetle/lighter/pull/61))
```

Comment:
Z tohoto napr. neviem vobec co sa stane :)

Reply:
Mohol si napr. cely support pre mdx zhrnut do jedneho bodu, bolo by to asi jednoduchsie na citanie.

Reply:
ok to som presne nevedel či to zhŕňať alebo nie. tak to skúsim zhrnúť všetko do jedného.